contrib/completion/bash/docker

```diff
@@ -334,14 +334,26 @@ _docker_diff() {
 
 _docker_events() {
 	case "$prev" in
-		--since)
+		--filter|-f)
+			COMPREPLY=( $( compgen -S = -W "container event image" -- "$cur" ) )
+			compopt -o nospace
+			return
+			;;
+		--since|--until)
+			return
+			;;
+	esac
+
+	case "${words[$cword-2]}$prev=" in
+		*event=*)
+			COMPREPLY=( $( compgen -W "create destroy die export kill pause restart start stop unpause" -- "${cur#=}" ) )
```

Comment:
We have some others that are somewhat similar though, don't we?  like `-v` completion with `:`?